config/webpack/environment.js

```diff
@@ -37,4 +37,31 @@ environment.loaders.append('yaml', {
   type: 'json'
 })
 
+/* The CDN url must be hardcoded at settings.yml:asset_host during assets compilation in order to get static assets
+ * like CSS files point to the CDN. Otherwise, the assets are generated with relative paths and are not loaded from
+ * the CDN, even when settings.yml:asset_host is set during runtime. We don't want to have to provide any CDN url when
+ * building porta docker images in order to get the assets correctly precompiled. To avoid that, this trick assumes
```

Comment:
```suggestion
 * building porta container images in order to get the assets correctly precompiled. To avoid that, this trick assumes
```

just Red Hat style guidelins